ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__nutanix-4.17-nightly-x86.yaml

```diff
@@ -0,0 +1,72 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-nutanix-417
+promotion:
```

Comment:
we don't need the images or promotion section here, its just making an un-needed postsubmit job

Reply:
Removed now,thanks!